static/app/views/issueList/actions/sortOptions.tsx

```diff
@@ -22,19 +21,14 @@ function getSortTooltip(key: IssueSortOptions) {
       return t('Number of events.');
     case IssueSortOptions.USER:
       return t('Number of users affected.');
-    case IssueSortOptions.TREND:
-      return t('% change in event count.');
     case IssueSortOptions.DATE:
     default:
       return t('Last time the issue occurred.');
   }
 }
 
-function getSortOptions(sortKeys: IssueSortOptions[], hasTrendSort = false) {
-  const combinedSortKeys = [
-    ...sortKeys,
-    ...(hasTrendSort ? [IssueSortOptions.TREND] : []),
-  ];
+function getSortOptions(sortKeys: IssueSortOptions[]) {
+  const combinedSortKeys = [...sortKeys];
   return combinedSortKeys.map(key => ({
```

Comment:
don't need this spread copy
```suggestion
  return sortKeys.map(key => ({
```

Reply:
Honestly this whole function could just go away and the map could just move to the `options` prop